Makefile

```diff
@@ -92,6 +92,14 @@ cluster-down:
 cluster-sync:
 	VERSION=$(VERSION) ./cluster/sync.sh
 
+cluster-sync: cluster-operator-push cluster-operator-install
+
+cluster-operator-push:
+	VERSION=$(VERSION) ./cluster/operator-push.sh
```

Comment:
You can replace VERSION=$(VERSION) for 
export VERSION := $(VERSION) in the top of the Makefile

Reply:
It is consistent with other targets, opened an issue to address it in another PR (no need to make this one even bigger) https://github.com/kubevirt/cluster-network-addons-operator/issues/168